fabric-data-generation-api-v1/src/main/java/net/fabricmc/fabric/api/datagen/v1/provider/FabricBlockLootTableProvider.java

```diff
@@ -72,7 +72,7 @@ public void excludeFromStrictValidation(Block block) {
 	}
 
 	@Override
-	public void accept(BiConsumer<Identifier, LootTable.Builder> biConsumer) {
+	public void accept(RegistryWrapper.WrapperLookup registryLookup, BiConsumer<Identifier, LootTable.Builder> biConsumer) {
```

Comment:
The same should be done with `FabricAdvancementProvider::generateAdvancement` as some advancement triggers now require `RegistryEntry`

Reply:
Im going to take this opportunity to just add it to them all, there is no reason not to